src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -2,0 +1,26 @@
+import { getUserById } from '../../App';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+import classNames from 'classnames';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const user = getUserById(todo.userId);
```

Comment:
Using `getUserById` here is not recommended as per the previous review. Instead, pass the user data as a prop from the App component to `TodoInfo` to adhere to the data flow guidelines.